virttest/test_setup.py

```diff
@@ -239,14 +240,14 @@ def __init__(self, test, params, session=None):
         while len(tmp_list) > 0:
             tmp_cfg = tmp_list.pop()
             test_cfg[re.split(":", tmp_cfg)[0]] = re.split(":", tmp_cfg)[1]
-        # Save host current config, so we can restore it during cleanup
+        # Save host current config
         # We will only save the writeable part of the config files
-        original_config = {}
+        current_config = {}
```

Comment:
Would you like to elaborate on why this local variable name had to be modified?

Reply:
I think 'original_config' is the original configuration of the test env, can be used to restore env,  but not only one instance can be created during the test, so i think it is more suitable to use 'current' when create new instance.